src/app/api/legal-db/route.ts

```diff
@@ -0,0 +1,168 @@
+import { CaseListing } from '@/types/schema';
```

Comment:
why are we creating an additional `api` folder in `app`? feels a bit confusing to have `src/api` and `src/app/api`

Reply:
NEVERMIND this is for routing...